src/main/java/org/opensearch/security/auditlog/impl/AuditMessage.java

```diff
@@ -463,15 +463,15 @@ public String getDocId() {
 	@Override
 	public String toString() {
 		try {
-			return Strings.toString(JsonXContent.contentBuilder().map(getAsMap()));
+			return org.opensearch.common.Strings.toString(JsonXContent.contentBuilder().map(getAsMap()));
```

Comment:
NOTE: this line and L474 are temporarily changed into this since the original `toString()` function is undefined in `org.opensearch.core.common.Strings`. I will create an issue to track that.

Reply:
Issue I created: https://github.com/opensearch-project/security/issues/2782